config/locales/en.yml

```diff
@@ -42,6 +42,8 @@ en:
     join: 'Join'
     leave: 'Leave'
     add: 'Add'
+    premade_add: 'Add Premade Content'
```

Comment:
We need to have Spanish translations for this in `es.yml` too.

We should change `premade_add: 'Add Premade Content'` to `suggested_content: 'Suggested Content'` and
`confirm_add: 'Confirm Add Content'` to `add_content: 'Add Content'`